### ISSUE TITLE
使用知识蒸馏框架遇到问题

### ISSUE THREAD

**Issue by liutheuniverse** [PaddlePaddle/Paddle#41962]
### 请提出你的问题 Please ask your question

环境：paddlepaddle2.2.2
我用的框架是这个https://aistudio.baidu.com/aistudio/projectdetail/3141968?channelType=0&channel=0
我想用自己的教师网络和数据库，目前教师网络已经打印输出，我在调用自己的数据库时，使用了这个代码替换了调用mnist的代码：
#训练数据加载
train_dataset = dataset('/home/aistudio/data',mode='train')
train_loader = paddle.io.DataLoader(train_dataset, batch_size=16, shuffle=True)
#评估数据加载
eval_dataset = dataset('/home/aistudio/data',mode='eval')
eval_loader = paddle.io.DataLoader(eval_dataset, batch_size = 8, shuffle=False)
然后我把后面的for batch_idx, data in enumerate(dataloader)改成了for batch_idx, data in enumerate(train_loader)
和for batch_idx, data in enumerate(eval_loader)
print不出来image和label的shape，然而调用数据库的代码cell却可以print出，两个地方隔了一个初始化代码，老师说数据库根本没有进入网络，我应该如何调整

**Comment by liutheuniverse** [comment]
[3609252.zip](https://github.com/PaddlePaddle/Paddle/files/8509582/3609252.zip)
代码

**Comment by liutheuniverse** [comment]
'int' object is not iterable
    5     acc_meter = AverageMeter()
      6     model.train()
----> 7     for batch_idx, data in enumerate(train_loader()):
      8         image = data[0]
      9         label = data[1]

**Comment by yaoxuefeng6** [comment]
看报错是数据输入有问题 建议排查下数据处理问题
